modules/client-configuration/pages/channels.adoc

```diff
@@ -42,3 +42,15 @@ In most cases, you will need to install the base channel for your distribution,
 
 If you need to transform Appstream repositories into non-modular repositories for use with some upgrade operations, you can use the Appstream filter with the content lifecycle management tool.
 For more information about the Appstream filter, see xref:administration:content-lifecycle-examples.adoc[].
+
+
+
+== Unified Installer Updates Channels on {sle} Clients
+
+All {sle} clients require access to an installer updates channel.
+This channel is used by the unified installer to ensure it is up to date before it installs the operating system.
```

Comment:
```suggestion
This channel is used by the unified installer to ensure it is up to date before it installs the operating system.
All {sle} proucts should have access to the installer updates channel during installation.
```

Reply:
Why reverse these sentences?

Reply:
The registered client does not need this channel. So I think it is more important to say that the unified installer need it for installing a client.
But more important is the change that the client does not need access to that channel, but all products should get it. If this is given, the rest should work out of the box.
You can change the order, but the change of the first sentence is important from my point of view. Otherwise we give a wrong impression.

Reply:
OK, updated.